### DISCUSSION TITLE
Tidak bisa menambahkan devices melalui postman

### DISCUSSION THREAD

**Discussion by azmirizkifar20** [kirimwa-id/forum#80] [Tanya Jawab] [answered]
Mohon maaf sebelumnya, saya ingin mencoba menambah device melalui postman namun terdapat kendala yaitu ketika mengirimkan request terdapat error "Error: Invalid character in header content ["Authorization"]"

![image](https://user-images.githubusercontent.com/43155964/148631042-4b54258d-9d45-4597-a79b-94eb6fdfe2ca.png)

apa karena character dari tokennya kah?

**Comment by fahram** [comment, marked as answer]
Di postmannya authorization typenya **Bearer Token** bukan ?

**Reply by azmirizkifar20** [reply]
sudah bisa mas @fahram, ternyata kalau dari postman memang gitu. tapi pas sudah di implement dan dicoba, alhamdulillah aman
